backend/tournesol/models/entity.py

```diff
@@ -163,6 +163,32 @@ def link_to_youtube(self):
             '<a href="https://youtu.be/{}" target="_blank">Play ▶</a>', self.video_id
         )
 
+    def criteria_scores_distributions(self, poll):
+        """Returns the distribution of critera score for the entities"""
+
+        # Fetch data with QuerySet
+        contributor_rating_criteria_score_list = [
+            [y for y in x.criteria_scores.all()]
+            for x in self.contributorvideoratings.filter(poll=poll)]
```

Comment:
This would trigger 1 SQL query per ContributorRating to fetch the criteria scores. It would be interesting to use `.prefetch_related()` here:

```suggestion
        contributor_rating_criteria_score_list = [
            [y for y in x.criteria_scores.all()]
            for x in self.contributorvideoratings.filter(poll=poll).prefetch_related(
                "criteria_scores"
            )
        ]
```